### ISSUE TITLE
mime-type

### ISSUE THREAD

**Issue by gregchalmers** [metocean/api-conventions#8]
https://tools.ietf.org/html/rfc2045#page-5
https://tools.ietf.org/html/rfc6838#page-7
"use of types in the "x." tree is strongly discouraged"

If this is public facing metocean should probably should use vendor mime types
examples:

application/vnd.metocean.grib2
application/vnd.metocean.netcdf4
application/vnd.metocean.cf+json

**Comment by aportagain** [comment]
I think this may have been resolved a while back by https://github.com/metocean/api-conventions/commit/692489c48d74cd712d0979d543a064a6112aa75b ... @gregchalmers, can you confirm that this is what you meant?

**Comment by gregchalmers** [comment]
@aportagain yep thats what I meant. I'm not sure what the existing applications are currently using thou.
